docs/lang/articles/get-started/index.md

```diff
@@ -5,7 +5,40 @@ slug: /
 
 # Getting Started
 
-Welcome to the Taichi Language documentation!
+## Target audience
+
+End users who *only* wish to quickly set up Taichi for simulation or high-performance numerical computation.
+
+:::caution IMPORTANT
+
+For developers who are interested in the compiler, computer graphics, or high-performance computing, and would like to contribute new features or bug fixes to the [Taichi programming language](https://github.com/taichi-dev/taichi), see the [Developer installation](../contribution/dev_install.md) for more information on building Taichi from source. Doing both at the same time may cause unnecessary conflicts.
```

Comment:
the sentence about "doing both" is outdated.

Reply:
Will also move this from dev install.